site-content/source/modules/ROOT/main-nav.adoc

```diff
@@ -10,6 +10,7 @@
 ** xref:community.adoc#meet-the-community[Meet the Community]
 ** xref:events.adoc[Events]
 * Learn
+** xref:site-content/source/modules/ROOT/pages/Apache-Cassandra-5.0-Moving-Toward-an-AI-Driven-Future.adoc[Cassandra 5.0]
```

Comment:
```suggestion
** xref:Apache-Cassandra-5.0-Moving-Toward-an-AI-Driven-Future.adoc[Cassandra 5.0]
```

???

Reply:
Shoot, maybe that was the issue, good catch. I'll try that now.

Reply:
what's the verdict ?

Reply:
i'm confused.

it is still (in `site-content/source/modules/ROOT/main-nav.adoc`)
```
xref:site-content/source/modules/ROOT/pages/Apache-Cassandra-5.0-Moving-Toward-an-AI-Driven-Future.adoc[Cassandra 5.0]
```

are you sure this is correct ?

Reply:
My fault, only fixed 1/2 the problem before. Amended now.